bin/update

```diff
@@ -119,6 +119,7 @@ foreach ($finder as $file) {
     }
 
     $contents = preg_replace('~{{>(\s*)(atoms|molecules|organisms)-~i', '{{>$1resources/templates/', $contents);
+    $contents = preg_replace('~{{\s*#_pl_only_([^ }]*).*/_pl_only_\1\s*}}~im', '', $contents);
```

Comment:
Try `~{{\s*#\s*_pl_only_([^ }]*).*?_pl_only_\1\s*}}~is`

Should fix some whitespace problems (eg new lines) and prevent greediness (in case it's used more than once).